beancount_import/source/ofx.py

```diff
@@ -967,10 +979,28 @@ def get_subaccount_cash(inv401ksource: Optional[str] = None) -> str:
 
                 cost_spec = None
                 price = None
-                is_sale = False
-                if raw.trantype in SELL_TYPES or (raw.trantype == 'TRANSFER' and
+                is_closing_txn = False
+                if raw.trantype in OPT_TYPES:
+                    assert(raw.optsharesperctrct is not None)
+
+                    if raw.opttradetype.find('TOCLOSE') == -1:
```

Comment:
Have you considered adding support for the `CLOSUREOPT` transaction type as well?
Not saying PR needs it, but it might be an easy addition.

Reply:
I don't know that I have a real example from real world ofx file, so I've only added code for transactions I've seen.